homeassistant/components/unifi/controller.py

```diff
@@ -122,6 +122,13 @@ def event_update(self):
             CONTROLLER_ID.format(host=self.host, site=self.site)
         )
 
+    @property
+    def event_options_update(self):
```

Comment:
Let's use the word `signal` because it's used within the dispatcher.

```suggestion
    def signal_options_update(self):
```